plugin-server/src/ingestion/ai-costs/providers/generated-providers.json

```diff
@@ -454,13 +454,6 @@
             "completion_token": 0.000005
         }
     },
-    {
-        "model": "gemini-pro-vision",
```

Comment:
Is this ok? Removing a model? Wont this break past spans?

Reply:
no its done on ingestion only